lametro/views.py

```diff
@@ -76,6 +76,10 @@ def extra_context(self):
         extra['bilingual'] = bool([e for e in extra['current_meeting'] if e.bilingual])
         extra['USING_ECOMMENT'] = settings.USING_ECOMMENT
 
+        upcoming = extra['upcoming_board_meetings']
```

Comment:
A couple of questions:

- What is returned from `LAMetroEvent.upcoming_board_meetings()`?
- We truncate whatever is returned to two items. Will this result in an incomplete list of upcoming meetings? (Are there ever more than two events on one day?)

Reply:
@hancush here are the answers to your questions:

* `LAMetroEvent.upcoming_board_meetings()` returns `LAMetroEvent` objects that contain the term 'Board Meeting' starting now or later.
* Yes, there are usually more than two events in one day. Based on a quick glance at Metro Legistar, it doesn't look like there are ever more than 2 events that contain the term 'Board Meeting' in a day, but I don't know of any reason why there couldn't be

Reply:
@fatima3558 Both of these answers suggest that `upcoming_board_meetings()` will omit meetings, i.e., it isn't a good solution to populate the new section. You want a list of all meetings with a start date of today. Can you propose a different solution?

Reply:
@hancush looks like the `is_upcoming` property is closer to what I need. It would include any meetings scheduled to start day or tomorrow (if the local time is after 5pm) up until the meeting starts. I would use this instead of `extra['upcoming_board_meetings']`

Reply:
@fatima3558 Does `is_upcoming` return a queryset of events meeting those criteria?

Reply:
I don't think that any of the existing properties are going to generate the list of events that you need. I'd like you to write a new property on the `LAMetroEvent` class that returns all meetings with a start date of the current date (being sure to calculate the current date in LA time, not UTC). Can you draft that, @fatima3558?

Reply:
ah, to answer your first question -- it returns a boolean, not a queryset

I can go ahead and draft a new property.

Reply:
Thanks, @fatima3558!